### PR TITLE
Result styling

### PR DESCRIPTION
**Pull request by karate**

### CODE REVIEW

File: dbSearcher/header.php [hunk 1]
```diff
@@ -6,20 +6,23 @@
         <link rel="stylesheet" href="css/styles.css" />
         <script>
             function copy(QueryId){
-                // Get all the querys made.
-                let querys = document.querySelectorAll("#query");
-
-                // Extract ours.
-                let temp = querys[QueryId].innerHTML;
+                // Get text of clicked element
```
Comment (PatricNox) [inline dbSearcher/header.php]: consistency, end comments with a dot if sentence

**Comment by PatricNox** [comment]
Nice update!

I also like the on-click auto copy, but I do think it's also better to keep the button as well due to user ability aspects.

**Comment by karate** [comment]
I was trying to get rid of the ugly and repeated html `<button>`.
I'll add the button as well and commit.

File: dbSearcher/header.php (hunk 1 above)
Comment (karate) [reply dbSearcher/header.php]: of course.

**Comment by PatricNox** [comment]
Screenshot gotta be updated after this too.

**Comment by karate** [comment]
Screenshot updated in  c050c10

**Pull request merged**